test-scripts/integrationTest.ts

```diff
@@ -31,7 +31,7 @@ async function setupVSCode(): Promise<string> {
         process.env.AWS_TOOLKIT_IGNORE_WEBPACK_BUNDLE = 'true'
 
         const disableExtensions = await getCliArgsToDisableExtensions(vsCodeExecutablePath, {
```

Comment:
should we even do this? seems pointless...

Reply:
I think this is mainly for running locally instead of via CI.

Reply:
as Bryce said, but also to be explicit about what is actually needed. Otherwise we have hidden dependencies